### PR TITLE
Add brad as resizer repo admin

### PR DESCRIPTION
**Pull request by gnufied**
Fixes https://github.com/kubernetes/org/issues/571

### CODE REVIEW

File: config/kubernetes-csi/org.yaml [hunk 1]
```diff
@@ -309,11 +309,13 @@ teams:
     description: admin access to external-resizer
     maintainers:
     - saad-ali
+    - childsb
```
Comment (nikhita) [inline config/kubernetes-csi/org.yaml]: oops...the list needs to be sorted :)

Comment (gnufied) [reply config/kubernetes-csi/org.yaml]: ha ha. okay!

**Comment by gnufied** [comment]
It looks to be failing on:

```
o: golang.org/x/lint@v0.0.0-20180702182130-06c8688daad7: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /home/prow/go/pkg/mod/cache/vcs/79f2695321bc515689ad880a6109a13399c003ff78670317e7d502240c43b765: exit status 128:
	fatal: remote error: Git repository not found
go: golang.org/x/text@v0.3.0: unknown revision v0.3.0
go: google.golang.org/api@v0.0.0-20181129220737-af4fc4062c26: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /home/prow/go/pkg/mod/cache/vcs/9e62a95b0409d58bc0130bae299bdffbc7b7e74f3abe1ecf897474cc474b8bc0: exit status 128:
	fatal: remote error: Git repository not found
go: google.golang.org/api@v0.0.0-20180910000450-7ca32eb868bf: unknown revision 7ca32eb868bf
go: error loading module requirements
```
Seems unrelated to this PR.

**Comment by nikhita** [comment]
:eyes: looks like a bad flake (came across this on https://github.com/kubernetes/org/pull/570 too)

/retest

**Pull request merged**